### DISCUSSION TITLE
Elevation of observing site

### DISCUSSION THREAD

**Discussion by Pilbarian** [Stellarium/stellarium#3883] [Q&A]
I was looking at the JPL Horizons manual and it discussed the allowance for the ellipsoid elevation due to variations around the world. 
I followed the instructions, went to the site offered to calculate the elevation and input coordinates for a number of random locations from Stellarium for comparison and found that some had negative elevation values. When looking at the total difference in elevations I found it could be as much as 80 metres. 
What do I do? Adjust the given results for geoid elevation, for known height above MSL and enter that into the location within Stellarium?  Or does Stellarium already include the geoid elevation variance in its given elevation levels? What about sites that are not in Stellarium’s list of observation locations?
I am wanting to experiment with archaeoastronomy on different locations but I imagine that large elevation differences may affect the results? 
Very new to this so I hope someone can assist. 
Cheers

**Comment by gzotti** [comment]
What do you want to achieve? Computation of topocentric parallax is based on the WGS84 ellipsoid. The mathematical horizon is 90 degrees from the zenith, regardless of whether you are in a geoid depression or on the highest mountain. Which results in archaeoastronomy do you think would change? Where are sub-arcseconds relevant? Atmospheric variance certainly creates more uncertainty. One crucial element is getting a correctly aligned horizon panorama. This is where I started in 2010... When you follow the User Guide, you can have that now. And much more.

**Reply by Pilbarian** [reply]
Thank you for your reply. 
I was just concerned that a large elevation difference may affect the potential alignment configuration. A shift in azimuth and elevation angles of the sun?
I think you are telling me that it will not be a problem, unless there was extreme precision needed?
Cheers

**Reply by Pilbarian** [reply]
This is what started me thinking about this …

“Altitude specifications should be with respect to the reference ellipsoid, not “mean sea level”. There can be more than 100 meters of difference between MSL and ellipsoid altitudes of a site. Separate on-line tools can help convert from MSL to ellipsoid altitude:

https://geographiclib.sourceforge.io/cgi-bin/GeoidEval”

From the JPL Horizons Manual, section on User defined topocentric coordinates. 
Thanks for your help. Appreciated.

**Reply by 10110111** [reply]
Ellipsoid is not geoid. Stellarium uses the WGS-84 ellipsoid, so I guess this is all you need.

**Reply by gzotti** [reply]
Geoid undulations also change the actual local zenith. I have not studied geodesy, though. It would be an interesting study how much the difference would really be and how much it would matter. Just take a few study locations (e.g. Stonehenge, Newgrange, Karnak (Egypt), Angkor Wat, Cuzco, anyplace on the Dead Sea shore, a few particularly oriented churches, ...) and compute whatever you are interested in, with location altitude values in MSL (Geoid) and WGS84 ellipsoid heights. When it comes to alignments on the ground, also investigate what changes when a stone may have moved after its placement. (Move its tip by 1 or 5cm. Does it change anything significantly?)   Do this computation with Stellarium (WGS84 ellipsoid), JPL Horizon (Geoid?), and e.g. Cartes du Ciel (?; or some other software titles). Apply both altitude values (although one will be false, of course). Do this computation for year 2000, and for year -2500, and tabulate the results. Then we could answer this question of how much this matters as compared to other sources of error.  I don't think such study was ever made public.

**Reply by Pilbarian** [reply]
Yes, 
“Geoid undulations also change the actual local zenith”

this is what I was thinking in relation to the geoid and its possible implications with astronomical alignments, certainly from a research perspective when not physically present at a particular location. I will experiment. 
Accurate data from a site would most likely be needed though in order to compare and correlate with the digital experience. 
There seems to be a deeper look happening with such sites as Stonehenge, beyond just a sunrise correlation, reflecting a much more accurate and comprehensive astronomical understanding by different cultures. Your work on Ollantaytambo is so interesting, the precision is quite astounding. 
Thanks for your help and ideas.

**Reply by gzotti** [reply]
Thanks, I still need to publish that in full! (Problem of "not critically urgent" things... You have seen the EAA talk then?) But yes, it shows what can be done with Stellarium in virtual archaeoastronomy, once you have a good 3D model to play with. (It's often difficult to get your hands on solid data and not be  restricted by NDAs.) But also here, you will not note any difference if you just raise the location by 100m. But if you are geodesist or are in a team with one, you could work on that topic.

**Reply by Pilbarian** [reply]
I am thinking that elevation will be more critical involving scenarios at low altitudes and, involving alignment of two or more points of interest/alignment due to azimuth and altitude shifting rapidly. A solar alignment at near vertical would be less affected by elevation variability than say a temple or other construct at sunrise/ sunset? Thoughts?
I have a fun project at hand that I have lidar data for, a geographically small area that is less affected by UTM distortion. Hoping to eventually create a 3D model for use in Stellarium, once I am confident that the theoretical alignments are actually present. 
Small elevation and azimuth angles at low altitude are very precise, but correct elevation relationships between multiple points are critical for arriving at a true result.
Ellipsoid model will give coordinates but accurate geoid elevations may be a critical factor. Fun in my spare time!

**Reply by gzotti** [reply]
The most crucial distortion from UTM coordinate is the meridian convergence offset. 
The rest is speculating. Don't "think", just measure and compute the differences in one particular location, then suggest viable improvements if necessary. Will be happy to read the resulting paper in an open-access journal.
